### PR TITLE
APEXCORE-536 #resolve Upgrade Hadoop dependency version from 2.2.0 to 2.6.0

### PR DESCRIPTION
**Pull request by davidyan74**

### CODE REVIEW

**Comment by davidyan74** [comment]
@tweise please merge if you're okay with the changes.

**Comment by vrozov** [comment]
Looks good. Please test on dev cluster.

**Comment by davidyan74** [comment]
Have tested pi demo with this on a cluster. It does not have a problem.

**Comment by davidyan74** [comment]
@tweise The voting has concluded and we can move forward now. Please merge when you have a chance.

File: engine/pom.xml [hunk 1]
```diff
@@ -388,6 +388,18 @@
...
     </dependency>
+    <dependency>
+      <groupId>com.sun.jersey.jersey-test-framework</groupId>
+      <artifactId>jersey-test-framework-grizzly2</artifactId>
+      <version>1.9</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
```
Comment (tweise) [inline engine/pom.xml:402]: Why is it required here but at the same time provided?

Comment (tweise) [reply engine/pom.xml:402]: If it is needed to make the test work, maybe add a comment why?

Comment (davidyan74) [reply engine/pom.xml:402]: this artifact is pulled in by hadoop-common:test and many other dependencies, but it conflicts with the artifact org.eclipse.jetty:jetty-servlet:jar during unit tests. That's why I needed to mark it provided so it won't be pulled in during the running of tests

Comment (davidyan74) [reply engine/pom.xml:402]: OK, will do

Comment (davidyan74) [reply engine/pom.xml:402]: done

**Pull request merged**